.github/workflows/book.rb

```diff
@@ -0,0 +1,21 @@
+require './item'
+
+# the book class
+class Book < Item
+  attr_accessor :publisher
+
+  def initialize(id, publisher, cover_state)
+    super()
+    @id = id || Random.rand(1...1000)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  private
+
+  def can_be_archieved?
+    return true if super || cover_state == 'bad'
+
+    false
+  end

```

Comment:
Great work on this project.

- [ ] But I think that the method code can be shorter than what you have here. See below my suggestion which does the same thing. 

Suggesting this 👇 
```
def can_be_archieved?
    super || cover_state == 'bad'   
 end
```